src/overlays/actors/ovl_Lamp_Light/ef_lamp_light.c

```diff
@@ -19,21 +25,114 @@ ActorProfile Lamp_Light_Profile = {
     /* */ Ef_Lamp_Light_actor_ct,
     /* */ Ef_Lamp_Light_actor_dt,
     /* */ Ef_Lamp_Light_actor_move,
-    /* */ (ActorFunc)none_proc1,
+    /* */ (void*)none_proc1,
     /* */ NULL,
 };
-#endif
+static s16 D_80967884_jp[4] = { 2, 2, 1, 0 };
```

Comment:
```suggestion
static s16 add_data_off[4] = { 2, 2, 1, 0 };
```

Is the static neccessary though

Reply:
they appear to be in function static in Ef_Lamp_Light_actor_move, so I would do that